homeassistant/components/fritz/services.py

```diff
@@ -32,7 +32,7 @@ async def async_call_fritz_service(service_call: ServiceCall) -> None:
         """Call correct Fritz service."""
 
         if not (
-            fritzbox_entry_ids := await _async_get_configured_avm_device(
+            fritzbox_entry_ids := await _async_get_configured_avm_wrapper(
```

Comment:
```suggestion
            fritzbox_entry_ids := await _async_get_configured_avm_device(
```

I think it makes sense to keep the "device" in the name